Readme.md

```diff
@@ -0,0 +1,95 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Chicken Meals!
+
+> In this project, we  will create a Meals application with javascript and API.
+
+
+
+<img src='https://github.com/Taher-web-dev/JS_Cap/blob/Dev/src/assets/Images/screenshoot.png' alt='screenshot'/>
+
+
+## Built With
+
+- HTML
+- CSS
+- git
+- Javascript
+- API
+
+
+## Live Demo
+
+
+[site](https://taher-web-dev.github.io/JS_Cap/)
```

Comment:
- [ ] Looks like the live demo link is not working perfectly. I am seeing this only (shown in the below screenshot). Please update the live demo link with the correct link or setup.


![Screenshot from 2021-11-18 23-53-59](https://user-images.githubusercontent.com/22434660/142474795-ba665363-a9e4-461e-963c-b1358b2e2c88.png)